<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addForm</class>
 <widget class="QWidget" name="addForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление контакта</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>351</width>
     <height>411</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>26</red>
         <green>249</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>26</red>
         <green>249</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>26</red>
         <green>249</green>
         <blue>234</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Данные контакта:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчество:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lFathername">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lFamily">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Никнейм:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lNick">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Город:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lCity">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ICQ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lICQ">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VK:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lVK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>http://vk.com/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lOK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>http://ok.ru/profile/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Facebook:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lFacebook">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>https://fb.com/profile.php?id=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>E-Mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lMail">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Дополнительная информация:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lPhone">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="butAdd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butClear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
